forge-gui-mobile/src/forge/adventure/pointofintrest/PointOfInterestChanges.java

```diff
@@ -41,7 +41,7 @@ public SaveFileData save() {
             }
             data.storeObject("keys",keys.toArray(new String[0]));
             for(int i=0;i<items.size();i++)
-                data.store("value_"+i,items.get(0).save());
+                data.store("value_"+i,items.get(1).save());
```

Comment:
@HassanSky81 why don't you update this?  You said in the comments it should be this:
```
data.store("value_"+i,items.get(i).save());
```

What's the correct method anyway @TrueFuFLeaderG

Reply:
(i) should be working but maybe it should be moved to SaveFileData

Reply:
I do not have access to my pc at the moment that is why I commented instead of changing. But I tested with "i" and it was remembering the bought cards after a save/load game. After Hollydays when I come home I will change it to "i" inside the pull request.

Reply:
You can also use Web IDE for small change